packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -645,18 +645,30 @@ class AnimationController extends Animation<double>
     _checkStatusChanged();
   }
 
-  /// Drives the animation with a critically damped spring (within [lowerBound]
-  /// and [upperBound]) and initial velocity.
+  /// Drives the animation with a spring (within [lowerBound] and [upperBound])
+  /// and initial velocity.
   ///
   /// If velocity is positive, the animation will complete, otherwise it will
   /// dismiss.
   ///
+  /// The default spring that is driven by this method is critically damped,
+  /// but the [springDescription] can be used to drive by a custom spring.
+  /// With this parameter you can make the fling slower, or otherwise faster.
```

Comment:
nit: I feel "fling slower" or "fling faster" is a bit inaccurate as it also depends on the initial velocity. How about something like:

The [springDescription] parameter can be used to specify a custom [SpringType.criticallyDamped] or [SpringType.overDamped] spring to drive the animation with. Defaults to null,  which uses a [SpringType.criticallyDamped] spring. See [SpringDescription.withDampingRatio] for how to create a suitable [SpringDescription].